app/components/search_result/collection/mods/metadata_component.rb

```diff
@@ -6,10 +6,12 @@ module Mods
       class MetadataComponent < Blacklight::Component
         def initialize(document:)
           @document = document
+          @fields = ['extent', 'summary_data']
```

Comment:
Can this be a constant?

Reply:
Yes but they are different across components so where/who are you suggesting it get defined?